gno.land/cmd/gnoland/testdata/unexpected-unreal.txtar

```diff
@@ -0,0 +1,75 @@
+# Reproducible Test for https://github.com/gnolang/gno/issues/1167
```

Comment:
I'm glad you tried txtar-based tests. How was your experience? Do you have any suggestions for improvement? If yes, please refer to #1269.

Reply:
This wasn't me, I just copied this: https://gist.github.com/thehowl/e36b0f0d652a2a348a2fcd331a310417. Added a comment there. We should avoid using environment variables for production. Other than that I'm liking it.

Reply:
My main thing with tests is this: 

I just want "make test" to work as expected, at the root, and at the individual project levels. At the root it should just pass any options and call make test individually. 

Anyone who pulls the codebase should be able to get make test passing, and go test ./... should all work as expected too.